packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -45,11 +45,13 @@ abstract final class SemanticsService {
   /// [1]: https://developer.android.com/reference/android/view/View#announceForAccessibility(java.lang.CharSequence)
   ///
   static Future<void> announce(
+    int viewId,
```

Comment:
This is a hard breaking change. A package that uses this API won't be able to support both a Flutter SDK before and after this change.

Instead, I would deprecate this API in favor of a new API which accepts a view ID. I'd consider introducing a `sendAnnouncement` API that accepts a view ID.